Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/CopyReplace4Colon.rdz.cbl

```diff
@@ -0,0 +1,10 @@
+﻿       IDENTIFICATION DIVISION.
```

Comment:
Test CopyReplace4Colon.rdz.cbl  is included in CopyReplace4Colon**4**.rdz.cbl  and can be removed.

Reply:
Tests CopyReplace4Colon4.rdz.cbl is here to test multiple COPY inclusion in a same file.

Reply:
Yes I understand that. 
My point is that test CopyReplace4Colon.rdz.cbl is useless as CopyReplace4Colon4.rdz.cbl already cover it.
Also the test for multiple COPY inclusion must be moved into TypeCobol.Test.
Here we are in a codegen test, so it assumes the parser already works as expected with `==::==`.
This test is only here to test the behavior of codegen.

Reply:
ok done https://github.com/TypeCobolTeam/TypeCobol/pull/1794/commits/4e0e1991fafc22e6a3291dbf91d2194ba9924e5e